__tests__/filterProperties.test.js

```diff
@@ -112,7 +112,7 @@ describe('filterProperties', () => {
     expect(filteredDictionary.properties).not.toHaveProperty('color');
   });
 
-  it('should work with a filter object', () => {
+  xit('should work with a filter object', () => {
```

Comment:
typo?

Reply:
It was a skipped test (this is how you tell Jest to skip a test). I had forgot to fix it 😬
Now I have updated it correctly for the way the function works now.

Reply:
Ohhh... TIL